src/script/api/script_goal.cpp

```diff
@@ -117,10 +117,12 @@
 	EnforcePrecondition(false, buttons < (1 << ::GOAL_QUESTION_BUTTON_COUNT));
```

Comment:
Needs a check whether buttons >= 0, otherwise `INT64_MIN` passes this test, but none of the required bits are set.